src/main/java/com/laytonsmith/core/events/Prefilters.java

```diff
@@ -129,25 +146,17 @@ public static void match(Map<String, Construct> map, String key,
 				case LOCATION_MATCH:
 					LocationMatch(map.get(key), actualValue);
 					break;
-            }
-        }
-    }
+			}
+		}
+	}
     
-    private static void ItemMatch(Construct item1, Construct item2) throws PrefilterNonMatchException{
-        String i1 = item1.val();
-        String i2 = item2.val();
-        if(item1.val().contains(":")){
-            String[] split = item1.val().split(":");
-            i1 = split[0].trim();
-        }
-        if(item2.val().contains(":")){
-            String[] split = item2.val().split(":");
-            i2 = split[0].trim();
-        }
-        if(!i1.trim().equalsIgnoreCase(i2.trim())){
-            throw new PrefilterNonMatchException();
-        }
-    }
+	private static void ItemMatch(Construct item1, Construct item2) throws PrefilterNonMatchException{
+		String i1 = item1.val().split(":")[0];
+		String i2 = item2.val().split(":")[0];
+		if (!i1.trim().equals(i2)) {
```

Comment:
i2 is always generated by the event, so I figured it didn't need trimming. Do I still need to trim it?